### PR TITLE
Lyke22

### PR DESCRIPTION
**Pull request by zachandfox**
[Issue #275](https://github.com/hivdb/covid-drdb-payload/issues/275)

### CODE REVIEW

**Comment by zachandfox** [comment]
Is there a way to differentiate subjects that receive a full dose booster and those that receive a half dose booster?

File: tables/vaccines.csv [hunk 1]
```diff
@@ -34,4 +34,5 @@ Ad26.COV2.S + mRNA-1273,Mixed,,,2,32
 Nanocovax,Protein subunit,Nanogen Pharmaceutical Biotechnology JSC,Vietnam,2,33
 BNT162b2 + CoronaVac,Mixed,,,2,34
 mRNA-1273.529,mRNA,Moderna,United States,2,35
-mRNA-1273 or mRNA-1273.529,mRNA,Moderna,United States,2,36
\ No newline at end of file
+mRNA-1273 or mRNA-1273.529,mRNA,Moderna,United States,2,36
+Ad26.COV2.S + BNT162b2,Mixed,,,2,37
```
Comment (KaimingTao) [inline tables/vaccines.csv:38]: @zachandfox if **Ad26.COV2.S + BNT162b2** is not used, please remove it. The subject_vaccine table I think is presenting the same meaning.

**Comment by KaimingTao** [comment]
> Is there a way to differentiate subjects that receive a full dose booster and those that receive a half dose booster?

**Comment by KaimingTao** [comment]
> Is there a way to differentiate subjects that receive a full dose booster and those that receive a half dose booster?

@zachandfox I didn't find **full dose**, **half dose** in the paper, do you have a screenshot?

**Comment by zachandfox** [comment]
In the main paper's abstract, pg 4:
Screenshot from 2022-05-03 11-25-03

In the supplementary document, supplementary tables: 
Screenshot from 2022-05-03 11-25-37

File: tables/vaccines.csv (hunk 1 above)
Comment (KaimingTao) [reply tables/vaccines.csv]: I see, sorry for my mistake.

**Comment by KaimingTao** [comment]
@

> In the main paper's abstract, pg 4: Screenshot from 2022-05-03 11-25-03
> 
> In the supplementary document, supplementary tables: Screenshot from 2022-05-03 11-25-37

Currently, we didn't have a single column to record the ug values, we may indicate in the __rx_name__, we may decide if we need the column later.

**Pull request merged**